### DISCUSSION TITLE
Is there any comparison between dkan1 and dkan2 regarding services and consumption?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3895] [Q&A]
Hello everyone, I hope you are well, I would like to know if there is any comparison of dkan1 vs dkan2 regarding consumption and services, or if perhaps dkan2 when using a new technology consumes much more compared to dkan1 for some specific reason such as in the import process Thanks a lot

**Comment by janette** [comment]
@tridoxx we do not have stats on a direct comparison between dkan v1 and dkan v2, but within dkan v2 is a module called [datastore_mysql_import](https://demo.getdkan.org/modules/contrib/dkan/docs/mysqlimport.html). This option provides a much faster import.

**Reply by tridoxx** [reply]
ty
